#This message allows the robot to send its omni vision current configuration
mirrorConfig mirrorConf
visionHSVConfig visionConf
